src/applications/disability-benefits/all-claims/pages/individualsInvolvedFollowUp.js

```diff
@@ -80,7 +80,7 @@ export const schema = index => ({
     [`incident${index}`]: {
       type: 'object',
       properties: {
-        personInvolved: {
+        personsInvolved: {
```

Comment:
This whole schema seems to match what's in `vets-json-schema`, aside from the added `view` property and the fact that the order is different. You can import that whole `personsInvolved` schema from there and use it here, and merge the `view` property into it. If you have trouble with the display ordering, you can use the `ui:order` uiSchema property

Reply:
It looks like the reason that we have this in the UI is because of the personsInvolved.name using the `fullName` definition, which we don't want.  I just noticed that this is an issue in the schema as well.  I'll update this in `vets-json-schema` so that this is possible.

Reply:
I've updated the personsInvolved name in vets-json-schema to make this do-able.

Reply:
What we've typically done is use the version in the shared schema and call out specific updates that need to happen. Something like:
```js
const personsInvolvedSchema = set('items.properties.rank.ui:collapsed', true, personsInvolved); // Pulled from vets-json-schema
personsInvolvedSchema = set('items.properties.unitAssigned.ui:collapsed', true, personsInvolvedSchema);
// etc.
```

I'm dreaming up writing a little `setMany` function to help with this, but it doesn't exist yet.

I see what you mean for the `name`. You don't want the suffix, right? That should be taken care of in the shared schema. It's fine if this overrides what's in the schema until you can get that fixed up, though. :+1:

:thinking: Although it looks like you removed the `name` schema override so it'll use the `fullName` definition. Was that intentional?